Common/MacrocosmUtils.cs

```diff
@@ -96,8 +97,28 @@ public class EnemyCategorization
         /// </summary>
         public static List<int> PlutoEnemies = new List<int>();
     }
-    public class RandomHelper
+    public static class RandomHelper
     {
+        public static object GetFieldValue(this Type type, string fieldName, object obj = null, BindingFlags? flags = null)
+        {
+            if (flags == null)
+            {
+                flags = BindingFlags.Instance | BindingFlags.Static | BindingFlags.NonPublic | BindingFlags.Public;
+            }
+            FieldInfo field = type.GetField(fieldName, flags.Value);
+            return field.GetValue(obj);
+        }
+
+        public static T GetFieldValue<T>(this Type type, string fieldName, object obj = null, BindingFlags? flags = null)
+        {
+            if (flags == null)
+            {
+                flags = BindingFlags.Instance | BindingFlags.Static | BindingFlags.NonPublic | BindingFlags.Public;
+            }
+            FieldInfo field = type.GetField(fieldName, flags.Value);
+            return (T)field.GetValue(obj);
+        }
```

Comment:
Why is this stuff in `RandomHelper`? This seems more like a `ReflectionHelper` to me.